jobs/JEVS_NFCENS_WAVE_GRID2OBS_STATS

```diff
@@ -43,12 +44,15 @@ export USHevs=${USHevs:-$HOMEevs/ush}
 ####################################
 # Define NET/RUN variables          
 ####################################
```

Comment:
@SamiraArdani-NOAA See previous comments about moving the entire "Define NET/RUN variables" section before the "Set EVS directories" section so that `export NET` stays in the Define NET/RUN variables" section. 

CC @PerryShafran-NOAA @ShelleyMelchior-NOAA